### ISSUE TITLE
[BUG] env-var-secret rule is triggered by `SECRETS_DIR`

### ISSUE THREAD

**Issue by jankatins** [stackrox/kube-linter#1198]
**System info:**
 - OS: linux

**Describe the bug**

In our helm chart, we declare a env var called `SECRETS_DIR`. This triggers the env-var-secret rule despite not containing a secret at all ut actually implementing the recommended strategy to mount the secrets.

**To Reproduce**
Declare a env var called `SECRETS_DIR`

**Sample YAML input**

```yaml
apiVersion: apps/v1
kind: Deployment
metadata:
apiVersion: apps/v1
kind: Deployment
metadata:
  name: whatever-service
  labels:
    app.kubernetes.io/managed-by: Helm
spec:
  replicas: 1
  template:
    spec:
      serviceAccountName: whatever-service
      initContainers:
        - name: db-migrations
        - command:
          - command
          env:
          - name: SECRETS_DIR  # <-
            value: /mnt/secrets
```

**Expected behavior**

I want that env var not to match the rule

**Screenshots**

Error:

```text
/chart/_pipeline_rendered_chart.yaml: (object: <no namespace>/whatever-service apps/v1, Kind=Deployment) environment variable SECRETS_DIR in container "db-migrations" found (check: env-var-secret, remediation: Do not use raw secrets in environment variables. Instead, either mount the secret as a file or use a secretKeyRef. Refer to https://kubernetes.io/docs/concepts/configuration/secret/#using-secrets for details.)
```

**Additional context**

Unfortunately, we do not have a way to pass in the ignore, as the actual env vars in the above resource definition are rendered by helm from values.yaml :-(
